src/Cake.Common/Tools/TextTransform/TextTransformAliases.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.TextTransform
+{
+    /// <summary>
+    /// Contains functionality related to transforming templates
+    /// </summary>
+    [CakeAliasCategory("TextTransform")]
+    public static class TextTransformAliases
+    {
+        /// <summary>
+        /// Transform a text template.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// // Transform a .tt template.
+        /// var transform = File("./src/Cake/Transform.tt")
+        /// TransformTemplate(transform, new TextTransformSettings { OutputFile="./src/Cake/Transform.cs" });
+        /// </code>
+        /// </example>
+        /// <param name="context">The context.</param>
+        /// <param name="sourceFile">The source file.</param>
+        /// <param name="settings">The settings.</param>
+        [CakeMethodAlias]
+        [CakeAliasCategory("TextTransform")]
```

Comment:
So I am clear int he actual Cake Alias Category it should read like this: `[CakeAliasCategory("T4TextTemplating")]`